app/api/collection/routes.py

```diff
@@ -6,7 +7,11 @@ def register_collection_role_api_urls(app):
     registrar = app.api_url_registrar
 
     registrar.register_get_routes(Collection, CollectionFacade)
-    registrar.register_post_routes(Collection, CollectionFacade)
+    registrar.register_post_routes(
+        Collection,
+        CollectionFacade,
+        [api_require_roles("contributor")]
```

Comment:
It may be the right time to add such new role requirements to PATCH and DELETE routes too.

Reply:
done; I also changed the role to admin as per Victor's request